ddtrace/bootstrap/sitecustomize.py

```diff
@@ -107,12 +182,17 @@ def update_patched_modules():
     if not opts:
         tracer.configure(**opts)
 
+    if will_run_module_cloning:
```

Comment:
The original comment you had was nice. Since the order of this executing does matter, right?

Are there any issues with moving it higher or lower? (semi-rhetorical question, just showcasing my thinking for why a comment could be nice to have)

Reply:
+1 on adding the comment back, just to explain why the order of execution does matter (maybe should also include why we don't care about the `from ddtrace import patch_all`)